README.md

```diff
@@ -0,0 +1,175 @@
+
+<div align="center">
+  <h3><b>Budget-App README</b></h3>
```

Comment:
- [ ] Kindly make sure your app has its own name related to the content as the previous reviewer requested.

## OPTIONAL]
- [ ] Kindly make sure next time you use the correct flow in the requirements of this project it's required to use gitflow, and for gitflow on solo projects the PR should be open from the feature branch to the dev branch